### PR TITLE
Review branch

### PR DESCRIPTION
**Pull request by carlylechia**
copied code from a trials repo so as to review

### CODE REVIEW

**Review by carlylechia — COMMENTED** [review]
You did a great job 👍🏽 However, there are a few things you can include in your code to make it even better. Check the comments under.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
```
Comment (carlylechia) [inline index.html:5]: I think it will be wise to give your project a concise and professional title, don't you?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            alt="the full microverse logo with a happy female developer in the background"
+            width="20%"
+            height="20%"
+        /></a>
```
Comment (carlylechia) [inline index.html:26]: I think you could do a better job with your space usage and indentation here. Make your code look easier on the eye. Go to a new line only when it is necessary.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        <b>Some Advantages of Microverse</b>
+        <li>It has very high standards</li>
+        <li>It is free until the developers get a job</li>
+        <li>It gives you a global community to work with</li>
+        <li>
+          It enables you to learn about new cultures while studying and working
+        </li>
+        <li>It is so fun and interesting</li>
+        <li>It prepares you for the global market</li>
+      </p>
```
Comment (carlylechia) [inline index.html:43]: It is semantically more appealing or correct to enclose <li> tags inside <ul> tags. here is an example:
<ul>
  <li>one</li>
  <li>two</li>
  <li>etc</li>
</ul>

File: index.html [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+        }
+        #img-caption {
+          width: 400px;
+        }
+        #image {
+          max-width: 100%;
+          height: auto;
+          display: block;
+        }
+      </style>
```
Comment (carlylechia) [inline index.html:58]: It is better to create a separate file for your styling and link it to your HTML file with the <lin> element up in the <head>.

**Review by carlylechia — COMMENTED** [review]
You did a great job 👍🏽 However, there are a few things you can include in your code to make it even better. Check the comments under.

**Review by ReshyResh — APPROVED** [review]
# Hi @carlylechia  , 

  - ✔Correct use of Github review comments.
  - ✔Comments are polite suggestions rather than orders.
  - ✔Explained why a change is required.

### Great job reviewing your past project! Your project is complete! 
#### Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**